README.md

```diff
@@ -4,29 +4,30 @@ copc-lib is a library which provides an easy-to-use reader and writer interface
 
 *Note: This library is in beta and API may change*
 
-## Installation
+## Build from source
 
 ### Dependencies
-copc-lib depends on laz-perf 2.1.0 or greater. See the [laz-perf repo](https://github.com/hobu/laz-perf) for installation instructions.
 
-### Build & Install
-With the dependencies installed, we can now build copc-lib. Note that you need to clone this repo with the `--recurse-submodules` flag, so that all dependencies get included.
+copc-lib has the following dependencies:
+- [laz-perf](https://github.com/hobu/laz-perf) >= 2.1.0
+- Catch2
+- Pybind11
 
 ### C++
 
 ```bash
-git clone --recurse-submodules https://github.com/RockRobotic/copc-lib.git
+git clone https://github.com/RockRobotic/copc-lib.git
 cd copc-lib
 mkdir build && cd build
 cmake ..
-make
-sudo make install
+cmake --build .
+sudo cmake --install .
 ```
 
 ### Python
 
 ```bash
-git clone --recurse-submodules https://github.com/RockRobotic/copc-lib.git
+git clone https://github.com/RockRobotic/copc-lib.git
 pip install ./copc-lib
 ```
 
```

Comment:
Should we provide instruction on how to run the example files in a different project?